domain/upgrade/state/state.go

```diff
@@ -301,53 +359,28 @@ func (st *State) ActiveUpgrade(ctx context.Context) (string, error) {
 	return activeUpgrade, errors.Trace(err)
 }
 
-// SetDBUpgradeCompleted marks the database upgrade as completed
-func (st State) SetDBUpgradeCompleted(ctx context.Context, upgradeUUID string) error {
+// UpgradeInfo returns the upgrade info for the provided upgradeUUID
+func (st *State) UpgradeInfo(ctx context.Context, upgradeUUID domainupgrade.UUID) (upgrade.Info, error) {
```

Comment:
`UpgradeInfo` vs `SelectUpgradeInfo`
Do we do this intentionally? Looks like the second one has a more explicit name.

Reply:
Yes, we should be consistent for getters. Select isn't required as much as get isn't.

Reply:
I have opinions on this. I'll air them in Riga.